content/usage/services.md

```diff
@@ -94,6 +94,10 @@ compose:
 
 For security reasons this option is only available to trusted repositories. Trusted repositories are enabled on a per-repository basis by a Drone administrator from your repository settings screen.
 
+Alternatively, each service will have access to the current source code under the ```/drone``` directory
```

Comment:
maybe link them back to the build workspace page for more information?
http://readme.drone.io/usage/build_test/#workspace:fb92aa3346185c57f15afda861d465a3